packages/components/cypress/component/bal-datepicker.cy.ts

```diff
@@ -95,4 +101,101 @@ describe('bal-datepicker.cy.ts', () => {
 
     cy.get('bal-datepicker').find('input.input').should('have.attr', 'disabled')
   })
+
+  it('should had disabled dates before min date', () => {
+    const onBalChangeSpy = cy.spy().as('balChange')
+
+    cy.mount(BalDatepicker, {
+      props: {
+        onBalChange: onBalChangeSpy,
+        min: '2023-01-10',
+      },
+    })
+
+    cy.get('bal-datepicker')
```

Comment:
- [ ] check first disabled and first enabled date
- [ ] check if you click a disabled cell that there is no change event to that date